livesync/indico_livesync/uploader.py

```diff
@@ -87,7 +87,6 @@ def processed_records(self, records):
             self.logger.debug('Marking as processed: %s', record)
             record.processed = True
         db.session.commit()
-        transaction.abort()  # clear ZEO cache
```

Comment:
Is the commit message correct? This doesn't seem to be related to livesync-invenio